nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,54 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 2.5.0
+***************
+
+:ref:`Core library <architecture>`
+==================================
+
+* The DFU result is now returned by the :c:param:`dfu_callback` callback in the :c:struct:`nrf_modem_init_params` struct.
+* It is no longer necessary to call :c:func:`nrf_modem_init` twice when applying modem firmware updates.
+* Added the SoftSIM API to use a software SIM with the nRF modem.
+
+:ref:`Sockets <nrf_sockets>`
+============================
+
+* Added
+
+  * The :c:macro:`NRF_SO_EXCEPTIONAL_DATA` socket option to enable sending data as part of exceptional events (3GPP).
+  * The :c:macro:`MSG_WAITACK` flag to request a blocking send operation until the request is acknowledged by the network.
+  * Enhanced APN rate control.
+
+* Removed:
+
+  * The :c:param:`sa_len`, :c:param:`sin_len` and :c:param:`sin6_len` from the :c:struct:`nrf_sockaddr` :c:struct:`nrf_sockaddr_in` and :c:struct:`nrf_sockaddr_in6` structs, respectively.
+
+* Replaced the :c:macro:`NRF_SO_BINDTODEVICE` socket option with :c:macro:`NRF_SO_BINDTOPDN`. The new option takes an integer for the PDN ID.
+
+:ref:`AT interface <nrf_modem_at>`
+==================================
+
+* Added the option to set a timeout for the time to wait for ongoing AT commands to complete by calling the :c:func:`nrf_modem_at_sem_timeout_set` function.
+
```

Comment:
We are missing one item here, the behavior of `nrf_modem_at_cmd_async` has changed slightly.

`nrf_modem_at_cmd_async` will now return immediately if there is another AT command pending.
Earlier, it only returned immediately if there was another AT command pending from an earlier `nrf_modem_at_cmd_async` call.

Perhaps we can phrase it like so:
`nrf_modem_at_cmd_async will now return immediately if there is another AT command pending, regardless of whether it was sent with nrf_modem_at_cmd_async or other API calls.`